pkg/authorization/cache/policy_cache.go

```diff
@@ -18,6 +18,18 @@ import (
 	bindingregistry "github.com/openshift/origin/pkg/authorization/registry/policybinding"
 )
 
+// PolicyCacheAdapter exposes getter methods for policy indexers and reflectors for the project cache
+type PolicyCacheAdapter interface {
+	// Return refrences to the indexers for the project cache to use
+	GetPolicyBindingIndexer() *cache.Indexer
+	GetPolicyIndexer() *cache.Indexer
+	GetClusterPolicyBindingIndexer() *cache.Indexer
+	GetClusterPolicyIndexer() *cache.Indexer
+
+	// Returns reflectors backing this cache for the project cache to use in determining synchronization skips
+	GetBackingReflectors() []*cache.Reflector
```

Comment:
Ideally, I would not need the reflector and instead I would want a token to represent the last observed state across each of the exposed indexers.  I think that we can do that in a follow-on though.